common/pulp/common/error_codes.py

```diff
@@ -114,6 +114,6 @@
 PLP1009 = Error("PLP1009", _("The request body does not contain valid JSON"), [])
 PLP1010 = Error("PLP1010", _("Provided value %(value)s for field %(field)s must be of type "
                              "%(field_type)s."), ["value", "field", "field_type"])
-PLP1011 = Error("PLP1011", _("Invalid task state passed to parameters list: %(state)s. "
-                             "Valid states are finished, error, timed out and skipped"), ["state"])
+PLP1011 = Error("PLP1011", _("Invalid task state passed to purge: %(state)s. "
+                             "Valid states are finished, error and/or skipped"), ["state"])
```

Comment:
Actually there is an issue. This message is used in API and CLI both. due to the task states discrepancy this message will fit only API or CLI. Maybe it will be better just to remove the part "Valid states are..."